src/common/http.cc

```diff
@@ -500,6 +500,18 @@ namespace Uri {
 
         return Some(it->second);
     }
+    
+    std::string 
+    Query::as_str() const {
+        std::string query_url;
+        for(const auto &e : params) {
+            query_url += "&" + e.first + "=" + e.second;
```

Comment:
This does not properly / safely escape e.first and e.second.

Pistache::Uri::Query::add() is public and can does not sanitize these values when adding them to the params map.  So it is incumbent on the caller to guarantee that they properly quote/escape 'name' and 'value'.  IMHO, this is not safe.

(I just read all of http.h and http.cc).  It appears that pistache does not URL query parameter sanitization or proper escaping / unescaping.  It seems that pistache requires the caller to pack/unpack quoted URIs.  So this code change is reasonable.  :(

https://en.wikipedia.org/wiki/Query_string

Reply:
xoac, I will merge this PR.  At first glance I was a bit worried about how the URL was constructed without escaping the values, then I see that Pistache never does that anyway.